tests/test_in_toto_record.py

```diff
@@ -105,11 +101,6 @@ def test_start_stop(self):
     self._test_cli_sys_exit(["stop"] + args + ["--products",
         self.test_artifact2, self.test_artifact2], 0)
 
-    # Start/stop recording verbosely
-    args = ["--step-name", "test4", "--key", self.key_path, "--verbose"]
-    self._test_cli_sys_exit(["start"] + args, 0)
-    self._test_cli_sys_exit(["stop"] + args, 0)
-
```

Comment:
Why did this test disappear?

Reply:
The test used to trigger an `if` statement in in_toto_record. That `if` statement was replaced by a logger method call `setLevelVerboseOrQuiet`, which I test separately in `test_log.py`.

Therefore, we are good in terms of coverage. In terms of interface testing, you are right, I could call the the cli tool with the verbose flag and test if it passes the right thing to `setLevelVerboseOrQuiet`. Not sure if necessary. What do you think?

Reply:
I don't think it's necessary. I just wanted to make sure you didn't delete that test for the sake of testing or something akin to that.